instrumentation/opentelemetry-instrumentation-flask/tests/test_programmatic.py

```diff
@@ -138,6 +140,8 @@ def setUp(self):
             "os.environ",
             {
                 "OTEL_PYTHON_FLASK_EXCLUDED_URLS": "http://localhost/env_excluded_arg/123,env_excluded_noarg",
+                "OTEL_PYTHON_HTTP_SERVER_REQUEST_DURATION_SERVER_ATTRIBUTES_ENABLED": "true",
```

Comment:
I think it's better to create a new bunch of tests for that, wdyt? For default, new_semconv and both_semconv